src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3,3 +3,19 @@ databaseChangeLog:
       file: db/changelog/changes/01-create-books-table.yaml
   - include:
       file: db/changelog/changes/02-create-users-table.yaml
+  - include:
+      file: db/changelog/changes/03-add-column-isDeleted-to-users-table.yaml
+  - include:
+      file: db/changelog/changes/04-create-roles-table.yaml
+  - include:
+      file: db/changelog/changes/05-insert-roles-into-roles-table.yaml
+  - include:
+      file: db/changelog/changes/06-create-users_roles-table.yaml
+  - include:
+      file: db/changelog/changes/07-insert-data-to-users-roles-table.yaml
+  - include:
+      file: db/changelog/changes/08-insert-more-data-to-users-roles-table.yaml
+  - include:
+      file: db/changelog/changes/09-drop-column-id-in-users-roles-table.yaml
+  #- include:
+  #   file: db/changelog/changes/10-add-constraints-to-users-roles-table.yaml
```

Comment:
Remove unnecessary comments

Reply:
i need that to pass mvn ckean package

Reply:
сomments to pass mvn clean package? I think there are some problems with your code if you need to have comments in changelog for mvn clean package

Reply:
i get Referential integrity violation when running tests but when i run applicatoin everething is fine. Our group student also have this problem and one of mentor said that it will be solved after migrating to Docker. i comment code that do not work with h2 and leave that works. if u know how to solve this pls tell me)

Reply:
please go to qna and we will try to resolve this :)